chapters/sampling-randomization-power.tex

```diff
@@ -1,427 +1,759 @@
+Stata has a built-in command called \texttt{ivregress},
+and another popular implementation is the user-written command \texttt{ivreg2}.
+
+
+%%%%% Matching
+
+\textbf{Matching}\sidenote{
+	\url{https://dimewiki.worldbank.org/Matching}}
+methods use observable characteristics to construct
+sets or pairs of treatment and control units
+so that the observations in each set are as similar as possible. \index{matching}
+These sets can either consist of exactly one treatment and one control observation (one-to-one),
+or it can be a set observations where either
+both groups have more than one observation represented (many-to-many),
+or where only one group has more than one observation included (one-to-many).
+By now you can probably guess that the result of the matching needs to be saved in the master dataset.
+This is best done by assigning an ID to each matching pair, 
+and create a variable in the master dataset with the ID for the pair each unit belongs to.
+
+The matching can even be done before the randomized assignment,
+so that treatment can be randomized within each matching set.
+This is a type of experimental design.
+Furthermore, if no control observations were identified before the treatment,
+then matching can be used to ex-post identify a control group,
+by finding the observations that are the most similar to the treated group.
+It is very difficult to test the validity of an ex-post matching
+as one would have to show that the difference in outcome is
+due to the impact of the treatment and not due to the groups not being similar enough.
+
+A valid matching must be made on data that is not related to the treatment
+or anything that the treatment could have affected in any way.
+Many matching algorithms can only match on a single variable,
+so you first have to turn many variables into a single varaible
+by using an index or a propensity score.\sidenote{
+	\url{https://dimewiki.worldbank.org/Propensity\_Score\_Matching}}
+The \texttt{iematch}\sidenote{
+	\url{https://dimewiki.worldbank.org/iematch}}
+command in the \texttt{ietoolkit} package developed by DIME Analytics
+produces matchings based on a single continuous matching variable.
+
+%-----------------------------------------------------------------------------------------------
+\subsection{Structuring datasets over multiple time periods}
+
+Your data map and master dataset requirements also depends on
+whether you are using data from one time period or several.
+A study that observes data in only one time period is called
+a \textbf{cross-sectional study}.
+\index{cross-sectional data}
+This type of data is relatively easy to collect and handle because
+you do not need to track individuals across time.
+Instead, the challenge in a cross-sectional study is to
+show that the control group is indeed a valid counterfactual to the treatment group.
+
+A study that observes data in multiple time periods is called \textbf{longitudinal}\index{longitudinal data}
```

Comment:
@bbdaniels , you said "check this" to _longitudinal_. Do you disagree? I am not sure what you mean.

Reply:
the grammar of this isn't quite right, (the study isn't called longitudinal data) -- perhaps: "Observations over multiple time periods, referred to as longitudinal data, can be either repeated cross-sections or panel data"

Reply:
ok, I see now. Fixed in cc2708116e4be7c887fe5bb4e57ef381d37a8ef9